infra/app/dapr-state-postgres.bicep

```diff
@@ -11,15 +11,22 @@ param tags object = {
   'azd-env-name': name
 }
 
-module daprBindingPGResources '../core/postgres.bicep' = {
+module daprBindingPGResources '../core/flexibleserver.bicep' = {
   name: 'dapr-binding-pg-${resourceToken}'
   params:{
-    name: name
+    name: 'pg${resourceToken}'
     location: location
     tags: tags
-    postgresUser: postgresUser
-    postgresPassword: postgresPassword
+    sku: {
+      name: 'Standard_B2s'
+      tier: 'Burstable'
+    }
+    storage: {
+      storageSizeGB: 64
```

Comment:
64 is closest option to what you were using, 50. I typically use the lowest, 32, for my azd samples. Is there a reason it needs higher?